corehq/apps/data_interfaces/app_config.py

```diff
@@ -0,0 +1,17 @@
+from django.apps import AppConfig
+
+from casexml.apps.case.models import CommCareCase
+from casexml.apps.case.signals import case_post_save
+
+from corehq.form_processor.models import CommCareCaseSQL
+from corehq.form_processor.signals import sql_case_post_save
+
+from corehq.apps.data_interfaces.signals import case_changed_receiver
+
+
+class DataInterfacesAppConfig(AppConfig):
+    name = 'data_interfaces'
+
+    def ready(self):
+        case_post_save.connect(case_changed_receiver, CommCareCase)
+        sql_case_post_save.connect(case_changed_receiver, CommCareCaseSQL)
```

Comment:
i've never done it this way before but i can't tell from reading [this](https://docs.djangoproject.com/en/1.10/ref/applications/#django.apps.AppConfig.ready) whether we should be importing models from other modules here or not. to be safe i think we may want to just use a string label for the sender rather than the model class